### PR TITLE
Replaced invalid categorical URIs for 8 parameters

### PR DESCRIPTION
**Pull request by junrongYu**
Fixes: #94

### CODE REVIEW

**Comment by edmondchuc** [comment]
The issue says the categorical collection's label cannot be fetched but this PR changes the URIs to the categorical collections. 

Are the labels not being fetched because those categorical collections no longer exist?

**Comment by junrongYu** [comment]
> The issue says the categorical collection's label cannot be fetched but this PR changes the URIs to the categorical collections.
> 
> Are the labels not being fetched because those categorical collections no longer exist?

hi @edmondchuc , yes, we deleted some duplicated categorical collections during the vocabulary migration, so some URIs became invalid.

File: vocab_files/observable_property_concepts/invertebrate-substrate-type.ttl [hunk 1]
```diff
@@ -14,7 +14,7 @@ making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
     skos:exactMatch <http://linked.data.gov.au/def/tern-cv/0209017c-b3eb-41bc-bf21-38c10c5ee063> ;
     skos:prefLabel "invertebrate substrate type" ;
```
Comment (edmondchuc) [inline vocab_files/observable_property_concepts/invertebrate-substrate-type.ttl:15]: How come this is invertebrate substrate type? Substrate type is only concerned with the soil.

Comment (edmondchuc) [reply vocab_files/observable_property_concepts/invertebrate-substrate-type.ttl:15]: This observable property is probably the same as https://linked.data.gov.au/def/test/dawe-cv/75f55bf0-6972-42ba-ad46-7e24f91e8f6a.

File: vocab_files/observable_property_concepts/lithology-type.ttl [hunk 2]
```diff
@@ -12,7 +12,7 @@ PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
     skos:exactMatch <http://linked.data.gov.au/def/tern-cv/e5b70249-76a8-4087-8ac9-588c7e29b944> ;
     skos:prefLabel "lithology type" ;
     schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/master/vocab_files/observable_property_concepts/lithology-type.ttl" ;
-    tern:hasCategoricalCollection <https://linked.data.gov.au/def/test/dawe-cv/46f50d33-eab0-4d00-9f84-58fd04017daf> ;
+    tern:hasCategoricalCollection <https://linked.data.gov.au/def/test/dawe-cv/1d50eb79-685f-45ea-84b4-627154eddede> ;
```
Comment (edmondchuc) [inline vocab_files/observable_property_concepts/lithology-type.ttl:15]: What's the difference between this `lithology type` observable property and `coarse fragments lithology`? They both share the same categorical collection.

File: vocab_files/observable_property_concepts/invertebrate-substrate-type.ttl (hunk 1 above)
Comment (junrongYu) [reply vocab_files/observable_property_concepts/invertebrate-substrate-type.ttl:15]: Yes, they are the same, thanks for clarifying that! How about creating another issue for this and fixing it in another PR?

Comment (edmondchuc) [reply vocab_files/observable_property_concepts/invertebrate-substrate-type.ttl:15]: Yeah sure.

File: vocab_files/observable_property_concepts/lithology-type.ttl (hunk 2 above)
Comment (junrongYu) [reply vocab_files/observable_property_concepts/lithology-type.ttl:15]: Hi Edmond, after reading the protocol, I think they should be the same property:

Description from `plot soil description`:

> Assess and record Surface coarse fragments, including their Abundance, Size, Shape, Lithology,
> Strength and Alteration using the codes in Appendix J. Record mulitple if present. Refer to pages 139–
> 144, 209 and 214 of the YB for further descriptions.

Description from `soil pit characterisation`:

> If present, assess and record Coarse fragments, including their Abundance, Size, Shape, Lithology,
> Strength and Alteration using the codes in Appendix T. Record mulitple if present. Refer to pages 139–
> 144, 209 and 214 of the YB for further descriptions.

They are the same, both the lithology type of coarse fragments. 

However, all other information is different, you can run this query to see:
```
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
PREFIX dcterms: <http://purl.org/dc/terms/>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
select ?moduleCollectionLabel ?parameterLabel ?parameterDefinition ?valueType ?featureType ?parameterMethod ?parameterSource ?categoricalCollectionLabel ?api
{
    ?moduleCollectionURI skos:member ?parameterURI ;
                         skos:prefLabel ?moduleCollectionLabel .
    ?parameterURI skos:prefLabel ?parameterLabel ;
    				tern:valueType ?valueTypeURI .
    ?valueTypeURI rdfs:label ?valueType .
    optional {?parameterURI skos:definition ?parameterDefinition .}
    optional {
            ?parameterURI tern:hasFeatureType ?featureTypeURI . 
            service <https://graphdb.tern.org.au/repositories/tern_vocabs_core> {
        ?featureTypeURI skos:prefLabel ?featureType .
    }}
    optional {
        ?parameterURI tern:hasMethod ?parameterMethodURI .
        ?parameterMethodURI skos:prefLabel ?parameterMethod .
    }
    optional {
        ?parameterURI dcterms:source ?parameterSource .
    }
    optional {
        ?parameterURI tern:hasCategoricalCollection ?categoricalCollectionURI .
        ?categoricalCollectionURI skos:prefLabel ?categoricalCollectionLabel ;
                                  dcterms:source ?api .
    }
    filter (?parameterLabel in ('lithology type', 'coarse fragments lithology'))
}
order by ?parameterLabel
```

I don't know how Arun interpret this property, should we discuss with him?

Comment (edmondchuc) [reply vocab_files/observable_property_concepts/lithology-type.ttl:15]: Yeah, we can confirm with Arun.

Comment (junrongYu) [reply vocab_files/observable_property_concepts/lithology-type.ttl:15]: Cool, I'll create another issue.

**Review by edmondchuc — APPROVED** [review]
Looks fine. Some follow-up issues have been created by June to address the unresolved comments in this PR. 

https://github.com/ternaustralia/dawe-rlp-vocabs/issues/122
https://github.com/ternaustralia/dawe-rlp-vocabs/issues/123

**Pull request merged**